requirements.txt

```diff
@@ -1,6 +1,8 @@
 pbr>=1.8
 netaddr>=0.7.18
-requests>=2.9.1
+requests
```

Comment:
I have to admit I'm not a Python developer so I tried to Google my way out of this issue. In between experiments, I was suggested to updated requests. Perhaps it was because it would require a newer version or urllib3 or because it already included a fix that urllib3 didn't had at the time of the writing of said suggestion.

All I can say, is that it works now!